app/models/bookmark.rb

```diff
@@ -106,6 +107,13 @@ def invalidate_bookmark_count
     end
   end
 
+  # We index the bookmark count, so if it should change, update the pseud
+  def update_pseud_index
+    return unless $rollout.active?(:start_new_indexing)
+    return unless destroyed? || saved_change_to_id? || saved_change_to_private?
```

Comment:
What case is `saved_change_to_id?` covering? Making a new bookmark?

Reply:
Yep. I don't think new_record? works after you save it.